apps/evm/src/hooks/useOperationModal/Modal/RepayForm/index.tsx

```diff
@@ -387,6 +392,27 @@ const RepayForm: React.FC<RepayFormProps> = ({ asset, pool, onCloseModal }) => {
     [isWrapUnwrapNativeTokenEnabled, asset.vToken.underlyingToken.tokenWrapped],
   );
 
+  const nativeTokenBalanceTokens = useMemo(() => {
+    return nativeTokenBalanceData
+      ? convertMantissaToTokens({
+          token: asset.vToken.underlyingToken.tokenWrapped,
+          value: nativeTokenBalanceData?.balanceMantissa,
+        })
+      : undefined;
+  }, [asset.vToken.underlyingToken.tokenWrapped, nativeTokenBalanceData]);
+
+  const shouldSelectNativeToken =
+    canWrapNativeToken && nativeTokenBalanceTokens?.gte(asset.userWalletBalanceTokens);
```

Comment:
This is actually a request from product, to select either the native token or the wrapped version, based on the balance the user has

Reply:
I see. Then my only suggestion would be to change this line to use `gt` instead `gte`, so that the underlying token is still favored over over the token being wrapped. What do you think?